lib/builtins/deploy-delegates/lambda-deployer/helper.js

```diff
@@ -272,3 +263,65 @@ function _updateLambdaFunction(reporter, lambdaClient, options, callback) {
         });
     });
 }
+
+function _updateLambdaFunctionCode(reporter, lambdaClient, options, callback) {
+    const { zipFilePath, deployState } = options;
+    const zipFile = fs.readFileSync(zipFilePath);
+    const functionName = deployState.lambda.arn;
+    const { revisionId } = deployState.lambda;
+
+    lambdaClient.updateFunctionCode(zipFile, functionName, revisionId, (err) => {
+        if (err) {
+            return callback(err);
+        }
+
+        reporter.updateStatus(`Update a lambda function code (${functionName}) in progress...`);
+
+        _waitForLambdaFunction(lambdaClient, functionName, callback);
+    });
+}
+
+function _updateLambdaFunctionConfig(reporter, lambdaClient, lambdaConfig, callback) {
+    const { FunctionName: functionName, Runtime: runtime, Handler: handler, RevisionId: revisionId } = lambdaConfig;
+
+    lambdaClient.updateFunctionConfiguration(functionName, runtime, handler, revisionId, (err) => {
+        if (err) {
+            return callback(err);
+        }
+
+        reporter.updateStatus(`Update a lambda function configuration (${functionName}) in progress...`);
+
+        _waitForLambdaFunction(lambdaClient, functionName, callback);
+    });
+}
+
+function _waitForLambdaFunction(lambdaClient, functionName, callback) {
+    const retryConfig = {
```

Comment:
nit: Could move this outside of the function.

Reply:
I just saw you approved this PR. Do you still need me to make this change?

I basically kept in line with the way the retry utility is used above.